internal/service/privatelinkendpointserviceserverless/resource_privatelink_endpoint_service_serverless_test.go

```diff
@@ -65,16 +129,17 @@ func checkDestroy(state *terraform.State) error {
 	return nil
 }
 
-func configBasic(orgID, projectName, instanceName, comment string) string {
+func configBasic(orgID, projectID, instanceName, comment string) string {
 	return fmt.Sprintf(`
 
-	resource "mongodbatlas_project" "test" {
-		name   = %[2]q
-		org_id = %[1]q
-	}
+	#resource "mongodbatlas_project" "test" {
```

Comment:
is this intentional?

Reply:
yess I was trying out if ProjectExecution causes any issues, cleaning up the PR, will update when RFAL

Reply:
cleaned up!